bulbs/content/models.py

```diff
@@ -117,6 +117,9 @@ def __unicode__(self):
         """
         return self.name
 
+    def is_new(self):
+        return not bool(self.pk)
```

Comment:
Not sure what the empty value is here, but I think you want to do something along the lines of:
```py
return self.pk is None
```